pom.xml

```diff
@@ -13,19 +13,12 @@
         Elucidation is a set of tools to allow for the visualization of the connections between microservices that make
         up an environment.
     </description>
-    <url>https://gitlab.com/fortitudetec/elucidation-project/elucidation</url>
+    <url>https://github.com/elucidation-project/elucidation</url>
     <inceptionYear>2018</inceptionYear>
 
-    <organization>
```

Comment:
Actually this should stay but just change name to "Elucidation Project" and URL to the GitHub URL, right? That's what we do in kiwi's parent POM